src/OrchardCore.Modules/OrchardCore.HealthChecks/Models/HealthCheckEntry.cs

```diff
@@ -0,0 +1,22 @@
+namespace OrchardCore.HealthChecks.Models;
+
+/// <summary>
+/// Represents a health check entry.
```

Comment:
These comments, and the ones on `HealthCheckReponse`, just repeat what's in the code. This is not very useful. Instead, add further details in the comment, examples, an explanation of how they are used in health checks, so one can understand the properties better.